packages/frontend/src/components/common/ActionCard.tsx

```diff
@@ -33,7 +33,7 @@ export const UpdatedInfo: FC<{
     updatedAt: Date;
     user: Partial<SessionUser> | undefined;
 }> = ({ updatedAt, user }) => {
-    const timeAgo = useTimeAgo(updatedAt);
```

Comment:
Why stop using the hook ? The hook makes sure the text keeps updating every X minutes. e.g if the user stays in the home page for 1 hour the label will update accordingly while your change would always display 'updated 1 minute ago'

Reply:
that hook was causing that the text was not updated inmediatelly, which is what this PR is trying to fix. 
Like in here: 

![Peek 2022-04-22 10-23](https://user-images.githubusercontent.com/1983672/164678075-5b0a9a8d-6c7a-4979-9e59-9206fccc876f.gif)
e

Reply:
I was under the impression the time wasn't updating because we weren't invalidating the `spaces` query. With your change `await queryClient.invalidateQueries('spaces');` it should be working as intended with the hook 🤔

Reply:
That's what I thought first, so I added it, but it was still not working, until removed the hook. Maybe it was a combination of both